doc/source/performance/procfs-plugin.rst

```diff
@@ -0,0 +1,139 @@
+.. procfs-plugin:
+
+======================================================================
+The ProcFS plugin
+======================================================================
+
+This feature is **tech preview** quality.
+
+Implemented in :rn:`8.0.25-15`, the ProcFS plugin provides access to the Linux performance counters by running SQL queries against a Percona Server for MySQL 8.0 or another MySQL 8.0-compatible variant. 
+
+In some environments, you are unable to capture operating system metrics , for example, Cloud installations or MySQL-as-a-service installations. These metrics are essential for complete system performance monitoring.
+
+The plugin does the following:
+
+* Reads selected files from the ``/proc`` file system and the ``/sys`` file system.
+
+* Populates the file names, and any content as rows in the `INFORMATION_SCHEMA.PROCFS`.
+
+The system variable :ref:`procfs_files_spec` provides access to the ``/proc`` and the ``/sys`` files and directories. This variable cannot be changed at run time, preventing a compromised account from giving itself greater access to those file systems. 
+
+
+Install the PLUGIN
+--------------------
+
+To install this plugin, copy the procfs.so file to the mysql plugin installation directory and execute the following command:
+
+.. sourcecode:: mysql
+
+
+    INSTALL PLUGIN procfs SONAME 'procfs.so';
+
+Access Privileges Required 
+ ---------------------------
+
+Only users with the `ACCESS_PROCFS` dynamic privilege can access the INFORMATION_SCHEMA.PROCFS view. During the plugin startup, this dynamic privilege is registered with the server. 
```

Comment:
Use RST syntax for monospaced font here:
```suggestion
Only users with the ``ACCESS_PROCFS`` dynamic privilege can access the ``INFORMATION_SCHEMA.PROCFS`` view. During the plugin startup, this dynamic privilege is registered with the server. 
```

Reply:
Added.